src/components/Explore/RightSidebar/GeneralSearch.tsx

```diff
@@ -0,0 +1,285 @@
+import React, { FC, ReactNode, useCallback, useContext, useEffect, useMemo, useState } from 'react';
```

Comment:
In this part, because of the changes you made in the map, When you select a particular station in the general filter, the behavior was to fly the map to those selected stations and gradually zoom out as the number of selected stations increased. Now, if you select stations in the general search, you will see that the map doesn't fly to the selected station even after you click search.